leda/components/Collapse/helpers.tsx

```diff
@@ -14,7 +14,8 @@ import { COMPONENTS_NAMESPACES } from '../../constants';
 import { LedaContext } from '../LedaProvider';
 import { Icon } from '../Icon';
 
-const defaultPanelWrapper = ({ children }: React.PropsWithChildren<{}>) => <>{children}</>;
+// eslint-disable-next-line react/jsx-no-useless-fragment
```

Comment:
You should properly configure this rule with allowExpressions set to true

https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-no-useless-fragment.md#allowexpressions

Reply:
What if I don't want to change this rule globally?